configurationfactory.go

```diff
@@ -214,7 +214,16 @@ func (arc *ConfigFactory) UnRegisterListener(listenerObj core.EventListener, key
 	return arc.dispatcher.UnRegisterListener(listenerObj, keys...)
 }
 
-// Unmarshal deserialize config into structure
+// Unmarshal function is used in the case when user want his yaml file to be unmarshalled to structure pointer
+//      (yaml files like chassis.yaml, lager.yaml,microservice.yaml)
```

Comment:
no need to say, archaius is a common lib

Reply:
delete this line    (yaml files like chassis.yaml, lager.yaml,microservice.yaml)